backend/app/routes/admin/getAdmins.js

```diff
@@ -0,0 +1,53 @@
+const to = require('await-to-js').default;
+const Admin = require('../../models/Admin');
+const { ErrorHandler } = require('../../../helpers/error');
+const constants = require('../../../constants');
+
+const createPipeline = (adminType, page) => {
+  const pipeline = [
+    {
+      $project: {
+        _id: 0,
+        firstName: '$firstName',
+        lastName: '$lastName',
+        username: '$username',
+        email: '$email',
+        contact: '$contact',
+      },
+    },
+  ];
+  if (adminType !== undefined) {
+    pipeline.unshift({
+      $match: {
+        isSuperAdmin: adminType === 'superAdmin',
+      },
+    });
+  }
+  if (page !== undefined) {

```

Comment:
pagination should be implemented by default. If the parameter is missing it is assumed to be page 1